virtcontainers/sandbox.go

```diff
@@ -60,15 +60,19 @@ type State struct {
 	// File system of the rootfs incase it is block device
 	Fstype string `json:"fstype"`
 
-	// Bool to indicate if the drive for a container was hotplugged.
-	HotpluggedDrive bool `json:"hotpluggedDrive"`
-
 	// PCI slot at which the block device backing the container rootfs is attached.
 	RootfsPCIAddr string `json:"rootfsPCIAddr"`
 
 	// Pid is the process id of the sandbox container which is the first
 	// container to be started.
 	Pid int `json:"pid"`
+
+	// Free static vCPUs that can be assigned to individual containers
+	FreeStaticCPU uint32 `json:"freeStaticCpu,omitempty"`
+
+	// Bool to indicate if the drive for a container was hotplugged.
+	// This is moved to bottom of the struct to pass maligned check.
```

Comment:
I dont think we need to document this in the code. Maybe just add it in the commit message.

Reply:
I was just moving the lines to pass CI memory alignment check.

Reply:
I added that line of comment so that in future someone wants to modify the struct, they know that the `Bool` one is better left at the bottom otherwise maligned check might fail.